linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_usermod/rule.yml

```diff
@@ -63,7 +55,4 @@ ocil: |-
 template:
     name: audit_rules_privileged_commands
     vars:
-        path: /usr/bin/usermod
-        path@ubuntu1604: /usr/sbin/usermod
-        path@ubuntu1804: /usr/sbin/usermod
-        path@ubuntu2004: /usr/sbin/usermod
+        path: /usr/sbin/usermod
```

Comment:
@brett060102 can you double check if this path is correct? I found it in the STIG to be `sbin`

Reply:
These can be changed back if needed.